packages/create-next-app/index.ts

```diff
@@ -388,6 +388,8 @@ async function run(): Promise<void> {
       if (ciInfo.isCI) {
         // We don't use preferences here because the default value is @/* regardless of existing preferences
         program.importAlias = defaults.importAlias
+      } else if (process.argv.includes('--no-import-alias')) {
+        program.importAlias = defaults.importAlias
```

Comment:
Currently our non-interactive tests specify the `--import-alias="@/*"` arg but this seems like a good alternative so the default doesn't need to be known. Could you update one of the tests doing the previous behavior with this instead?

x-ref: https://github.com/vercel/next.js/blob/ea65cb6541fa5c675e50f56561c093dfe095fbcb/test/integration/create-next-app/templates-app.test.ts#L136